lms/djangoapps/course_wiki/middleware.py

```diff
@@ -55,6 +57,21 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # lint-amne
         course_id = course_id_from_url(request.path)
         wiki_path = request.path.partition('/wiki/')[2]
 
+        # if no wiki_path, can't get wiki_slug, so no point trying to look up
+        # course_id by wiki_slug
+        if not course_id and wiki_path:
+            # wiki path always starts with wiki_slug
+            wiki_slug = wiki_path.split('/')[0]
+
+            modstore = modulestore.django.modulestore()
+            course_ids = modstore.get_courses_for_wiki(wiki_slug)
```

Comment:
Tiny nit: usually, we just import `from xmodule.modulestore.django import modulestore` and then do things like `course_ids = modulestore().get_courses_for_wiki(wiki_slug)` directly.

Reply:
@Agrendalath Ok, I'll update it now.

Reply:
Done.